lib/linter.js

```diff
@@ -63,7 +58,9 @@ class Linter {
 
         const tree = this.parse(source);
 
-        this.ast = tree.root.toResult();
+        this.ast = tree.root.toResult({
+            stringifier: stringify
```

Comment:
@shellscape I'm not 100% sure about this. Without it, it fails with 

```
TypeError: this[node.type] is not a function
    at Stringifier.stringify (/Users/jwilsson/lesshint/node_modules/postcss/lib/stringifier.js:33:24)
    at Stringifier.body (/Users/jwilsson/lesshint/node_modules/postcss/lib/stringifier.js:93:18)
    at Stringifier.root (/Users/jwilsson/lesshint/node_modules/postcss/lib/stringifier.js:37:14)
    at Stringifier.stringify (/Users/jwilsson/lesshint/node_modules/postcss/lib/stringifier.js:33:24)
    at MapGenerator.stringify (/Users/jwilsson/lesshint/node_modules/postcss/lib/stringify.js:14:9)
    at MapGenerator.generate (/Users/jwilsson/lesshint/node_modules/postcss/lib/map-generator.js:300:18)
    at LazyResult.stringify (/Users/jwilsson/lesshint/node_modules/postcss/lib/lazy-result.js:294:24)
    at Root.toResult (/Users/jwilsson/lesshint/node_modules/postcss/lib/root.js:109:21)
    at new Linter (/Users/jwilsson/lesshint/lib/linter.js:61:30)
    at Lesshint.checkString (/Users/jwilsson/lesshint/lib/lesshint.js:108:28)
```

I don't know if that's expected or not? If this line is required that's of course fine, just wanted to check in with you.